src/app.py

```diff
@@ -161,21 +180,26 @@ async def spawn_process_for_new_agent(object: Object):
     await parent_connection.coro_send(state.scene.serialize())
     state.processes[object.id] = ProcessData(object_id=object.id, process=process, pipe=parent_connection)
 
+async def create_tick():
+    while True:
+        await tick_trigger()
+        await asyncio.sleep(1 - missing_to_full_second())
+
 
 #  __   __       ___  __  __
 # |__) /  \ |  |  |  |__ (__'
 # |  \ \__/ \__/  |  |__ .__)
 
 
 @app.get("/")
-async def get(tick_number: int, background_tasks: BackgroundTasks) -> Dict[str, Union[int, str, List[List[str]]]]:
-    background_tasks.add_task(tick_trigger)
+async def get(tick_number: int) -> Dict[str, Union[int, str, List[List[str]]]]:
     # if player has no scene or player tick number is incorrect
+    state.available_ticks_without_get = 3
```

Comment:
Can you please explain what this exactly is? It's a bit difficult to tell by the name

Reply:
Idk, random number which imo is ok. Because when we don't `get` ,we don't  want map to change (tick)

Reply:
I think it can be a `bool` then.
When someone sends a `get` request: `should_tick = true`
After tick: `should_tick = false`

Reply:
Nah, because then we will back to situation in which we were before.

Reply:
Why do you think so? I can't imagine that

Reply:
When we change boolean to false on the end of tick, then another get must turn it on, so we are depended on `gets`

Reply:
`state.available_ticks_without_get = 3` is exactly the same, then.
In this case, after around one second ticking must be turned on again, so it is dependent ot `get`s as before

Reply:
Fair, but when we have higher margin, even when we have lag/skip one `get`, then map still works in the background.